packages/@react-spectrum/menu/stories/MenuTrigger.stories.tsx

```diff
@@ -527,6 +528,47 @@ storiesOf('MenuTrigger', module)
         <Item key="3">Three</Item>
       </Menu>
     )
+  ).add(
+    'with Selection Group',
+    () => render(
+      <Menu onAction={action('onAction')} disabledKeys={["8"]}>
+        <Section title="Single selection">
+          <SelectionGroup selectionMode="single">
+            <Item key="1">
+              One
+            </Item>
+            <Item key="2">
+              Two
+            </Item>
+            <Item key="3">
+              Three
+            </Item>
+          </SelectionGroup>
+        </Section>
+        <Section title="Multi selection">
+          <SelectionGroup selectionMode="multiple">
+            <Item key="7">
+              Seven
+            </Item>
+            <Item key="8">
+              Eight
+            </Item>
+            <Item key="9">
+              Nine
+            </Item>
+          </SelectionGroup>
+        </Section>
+        <Item key="4">
```

Comment:
It's confusing that the section/selectiongroup has ended, not a fault of your PR, it's working pretty well, just noting something we'll need to solve for
I'll bring it up with design and see if we can't get some guidelines around mixing Sections with Items and what SelectionGroups can go around. We may decide to handle that in follow-up work.